style.css

```
@@ -1,7 +1,7 @@
 body {
 	margin: 0px;
 	font-family: Helvetica;
-
+	font-size: 12px;
```

I think you messed up slightly. It should have been 18px.
```suggestion
	font-size: 18px;
```